Agent and Backend Ref for Env Vars (#1724)
sensu/sensu-docs: content/sensu-go/5.12/reference/backend.md

  - [Dashboard configuration](#dashboard-configuration-flags)
  - [Datastore and cluster configuration](#datastore-and-cluster-configuration-flags)
  - [Advanced configuration options](#advanced-configuration-options)

>   - [Configuration via environment variables](#configuration-via-environment-variables)

Reviewer: I believe that these env vars will be supported in the backend starting with the 5.15 release, so let's move this change into that version of the content. It should not be present in versions prior to 5.15.

>   - [Configuring Flags as Environment Variables](#configuring-flags-as-environment-variables)

Reviewer: ```suggestion
  - [Configuration via environment variables](#configuration-via-environment-variables)
```

# /etc/sensu/backend.yml example
event-log-buffer-size: 100000{{< /highlight >}}

> ### Configuring Flags as Environment Variables

Reviewer: ```suggestion
### Configuration via environment variables
```

> The `sensu-backend` service can read environment variables from `/etc/default/sensu-backend` on Debian/Ubuntu systems and `/etc/sysconfig/sensu-backend` on RHEL systems. These files are not created by the installation package and will need to be created.

Reviewer: ```suggestion
The `sensu-backend` service configured by our supported packages will read environment variables from `/etc/default/sensu-backend` on Debian/Ubuntu systems and `/etc/sysconfig/sensu-backend` on RHEL systems. These files are not created by the installation package and will need to be created.
```

{{< language-toggle >}}

{{< highlight "Ubuntu/Debian" >}}
$ sudo touch /etc/default/sensu-backend
{{< /highlight >}}

{{< highlight "RHEL/CentOS" >}}
$ sudo touch /etc/sysconfig/sensu-backend
{{< /highlight >}}

{{< /language-toggle >}}

> For any configuration flag you wish to specify as an environment variable, you will need to append `sensu_` and convert dashes (`-`) to underscores (`_`). Then, add the resulting environment variable to the appropriate environment file described above. You must restart the service for these settings to take effect.

Reviewer 2: ```suggestion
For any configuration flag you wish to specify as an environment variable, you will need to append `SENSU_` and convert dashes (`-`) to underscores (`_`). Then, add the resulting environment variable to the appropriate environment file described above. You must restart the service for these settings to take effect.
```

> In the following example, the `API_LISTEN_ADDRESS` flag is configured as an environment variable and set to `192.168.100.20:8080`.

Reviewer: ```suggestion
In the following example, the `api-listen-address` flag is configured as an environment variable and set to `192.168.100.20:8080`.
```
